### PR TITLE
Add replace-ext

### PR DESCRIPTION
**Pull request by DeividasBakanas**
NPM package: https://www.npmjs.com/package/replace-ext

### CODE REVIEW

File: replace-ext/replace-ext.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Type definitions for replace-ext 0.0.1
+// Project: https://www.npmjs.com/package/google-maps
```
Comment (vvakame) [inline replace-ext/replace-ext.d.ts]: use `https://github.com/wearefractal/replace-ext`

File: replace-ext/replace-ext.d.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+// Definitions by: Deividas Bakanas <https://github.com/DeividasBakanas>, Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
```
Comment (vvakame) [inline replace-ext/replace-ext.d.ts]: is this right?

**Comment by DeividasBakanas** [comment]
Comments resolved and waiting for review.

**Comment by horiuchi** [comment]
@DeividasBakanas Thanks mate!

**Pull request merged**